src/nix/run.md

```diff
@@ -80,6 +80,7 @@ An app is specified by a flake output attribute named
 apps.x86_64-linux.blender_2_79 = {
   type = "app";
   program = "${self.packages.x86_64-linux.blender_2_79}/bin/blender";
+  description = "Run Blender, a free and open-source 3D creation suite.";
```

Comment:
I guess this should be `meta.description`?

However I'm not sure we need the `meta` prefix. It's necessary for derivations to distinguish derivation input attributes from metadata, but that's not the case for `apps`.

Reply:
Though I suppose `meta` is useful for distinguishing between attributes that should be an error if they're unknown, and attributes that should at most be a warning.

Reply:
I think that's a good argument.
Also users are likely to write `meta` because they learned it for packages, and it's nice that `x.meta.description or ""` works for both, if that's going to be relevant.